docs/ovsa/ovsa_get_started.md

```diff
@@ -314,7 +314,7 @@ Begin these steps on the Host Machine.
 
 As an option, you can use `virsh` and the virtual machine manager to create and bring up a Guest VM. See the `libvirtd` documentation for instructions if you'd like to do this.
 
-1. Download the [Ubuntu 18.04 server ISO image](https://releases.ubuntu.com/18.04/ubuntu-18.04.5-live-server-amd64.iso)
+1. Download the **Server install image for 64-bit PC (AMD64) computers** (ISO) of [Ubuntu 18.04](https://releases.ubuntu.com/18.04/)
```

Comment:
Will releases higher than 18.04.5 be supported? Right now the aforementioned version isn't available on the provided website.